plugins/inputs/prometheus/prometheus.go

```diff
@@ -36,6 +36,10 @@ type Prometheus struct {
 
 	ResponseTimeout internal.Duration `toml:"response_timeout"`
 
+	MetricVersion int `toml:"metric_version"`
+
+	UrlTag string `toml:"url_tag"`
```

Comment:
Just a naming nitpick, can you name this variable `URLTag` so that it matches the Go recommendations for [initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).

Reply:
fixed this.